samples/PushDescriptors/CMakeLists.txt

```diff
@@ -40,6 +32,5 @@ add_executable(PushDescriptors
 )
 
 set_target_properties(PushDescriptors PROPERTIES FOLDER "Samples")
-target_compile_definitions(PushDescriptors PUBLIC VULKAN_HPP_DISPATCH_LOADER_DYNAMIC=1)
-target_include_directories(PushDescriptors PUBLIC ${CMAKE_SOURCE_DIR}/glslang)
-target_link_libraries(PushDescriptors PUBLIC glslang SPIRV)
+target_link_libraries(PushDescriptors PRIVATE utils)
+target_link_libraries(PushDescriptors PRIVATE "${Vulkan_LIBRARIES}")
```

Comment:
Why are the Vulkan_LIBRARIES needed here?

Reply:
Seems mistake. I'll look into it.

Reply:
Looks like following line causes linker error without `Vulkan_LIBRARIES`
```   
 commandBuffer->pushDescriptorSetKHR(vk::PipelineBindPoint::eGraphics, *pipelineLayout, 0, { 2, writeDescriptorSets }, vk::DispatchLoaderDynamic(*instance, *device));
```


It calls `vk::DispatchLoaderDynamic(*instance, *device)` which ends up accessing `::vkGetInstanceProcAddr` and `::vkGetDeviceProcAddr `.

Reply:
Seems, there's an error in that call: one should not use that temporary vk::DispatchLoaderDynamic. That has somehow survived my changes on using the DispatchLoaderDynamic as the default one.
Could you please make sure that you still need Vulkan_LIBRARIES, if you remove that function argument?

Reply:
Sure. Fixed in e0a026a.